recipes-core/systemd/files/0001-systemd-networkd-wait-online.service-wait-for-any-ne.patch

```diff
@@ -0,0 +1,30 @@
+From dc0998b6aae735c697a3a64b6b9108d08c24abaf Mon Sep 17 00:00:00 2001
+From: Roger Luethi <roger.luethi@bisdn.de>
+Date: Mon, 25 Jul 2022 11:37:14 +0200
+Subject: [PATCH] systemd-networkd-wait-online.service: wait for any network
+ interface
+
+Wait for any network interface to come up (instead of waiting for all of
+them).
+
+Signed-off-by: Roger Luethi <roger.luethi@bisdn.de>
+---
+ units/systemd-networkd-wait-online.service.in | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/units/systemd-networkd-wait-online.service.in b/units/systemd-networkd-wait-online.service.in
+index 10d8b08c8e..6dd5f79377 100644
+--- a/units/systemd-networkd-wait-online.service.in
++++ b/units/systemd-networkd-wait-online.service.in
+@@ -18,7 +18,7 @@ Before=network-online.target shutdown.target
+ 
+ [Service]
+ Type=oneshot
+-ExecStart={{ROOTLIBEXECDIR}}/systemd-networkd-wait-online
++ExecStart={{ROOTLIBEXECDIR}}/systemd-networkd-wait-online --any
```

Comment:
i think we want to specifically wait for the management interface here

Reply:
I considered that, but I don't know a good method of finding the management interface. We could change the installer to rename a likely looking interface ma1 (something like what ONL does).

Alternatively, we can patch docker.service to not wait for network-online.target, but then it won't wait even for the first interface to come up.

Reply:
i think it might be good to use a systemd drop in here

Reply:
Good idea. Done.